life/modernized/04-part-2-section-1.adoc

```diff
@@ -2,16 +2,8 @@
 
 === Section I.
```

Comment:
Sorry about that! I didn't notice the dashes and thought it was more of an intro.

Reply:
Yes, I started the PR for italics but I remembered Jared telling me that it's better to wait to work on chapters in a new PR that haven't been merged in another one yet, I believe. So that's why I started on Edmundson in the mean time. :)